### PR TITLE
Updated notebook section of participate.md

### PR DESCRIPTION
**Pull request by nicole-brewer**
## Description
Updated notebook section of participate.md and added a link to the submission template Google doc.

## Checklist:

- [ ] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2024-conference-planning-committee) to ask for content reviewers
- [ ] I have previewed changes locally

cc @jrgochan

### CODE REVIEW

File: pages/participate/participate.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Papers will consist of submissions up to 10 pages in length, including figures,
...
... might aim to showcase tools through engaging narratives or release interactive dashboards for exploring datasets related to RSE careers. To account for the broad variety of use cases for notebooks, we officially support submissions in R (i.e. Rmd, Qmd); Python (i.e. ipynb, Qmd); or other languages [supported by Quarto](https://quarto.org/docs/faq/#what-programming-languages-are-supported-in-quarto), but we can also accept any language supported by  and [its backend engines](https://quarto.org/docs/computations/execution-options.html#engine-binding) which include [Jupyter kernels](https://docs.jupyter.org/en/latest/projects/kernels.html) and the [Knitr engine](https://bookdown.org/yihui/rmarkdown-cookbook/other-languages.html). Examples of notebook submissions may include, but not limited to, the following: showcasing a tool, workflow, or dataset, or publishing a notebook-based research/data paper, lesson, or other instructional document. Accepted notebooks will be given the opportunity for a 12-minute oral presentation plus 3 minutes for questions. [Notebook submission template here](https://docs.google.com/document/d/16pGPdGWpyOr79pgBxEImAHioPgyy-jJwiIklJUJWE_4/edit?usp=sharing).
```
Comment (danielskatz) [inline pages/participate/participate.md:68]:
```suggestion
There are many reasons RSEs might want to publish a computational notebook. For example, they might aim to showcase tools through engaging narratives or release interactive dashboards for exploring datasets related to RSE careers. To account for the broad variety of use cases for notebooks, we officially support submissions in R (i.e., Rmd, Qmd), Python (i.e., ipynb, Qmd), and other languages [supported by Quarto](https://quarto.org/docs/faq/#what-programming-languages-are-supported-in-quarto), but we can also accept any language supported by [its backend engines](https://quarto.org/docs/computations/execution-options.html#engine-binding), which include [Jupyter kernels](https://docs.jupyter.org/en/latest/projects/kernels.html) and the [Knitr engine](https://bookdown.org/yihui/rmarkdown-cookbook/other-languages.html). Examples of notebook submissions may include, but not limited to, the following: showcasing a tool, workflow, or dataset, or publishing a notebook-based research/data paper, lesson, or other instructional document. Accepted notebooks will be given the opportunity for a 12-minute oral presentation plus 3 minutes for questions. [Notebook submission template here](https://docs.google.com/document/d/16pGPdGWpyOr79pgBxEImAHioPgyy-jJwiIklJUJWE_4/edit?usp=sharing).
```

Comment (lmilechin) [inline pages/participate/participate.md:68]: Template document at link is still titled "Notebooks Submission Template Draft". Also be sure to make the template readable to everyone once released (right now access is restricted).

Comment (torwhite) [reply pages/participate/participate.md:68]: Thanks! Will do!

Comment (torwhite) [reply pages/participate/participate.md:68]: Thanks Dan, the grammar tweaks look good to me.

**Review by jsubida — APPROVED** [review]

**Pull request merged**